packages/angular_devkit/build_webpack/src/browser/index.ts

```diff
@@ -150,6 +152,12 @@ export class BrowserBuilder implements Builder<BrowserBuilderOptions> {
           return;
         }
         const webpackCompiler = webpack(webpackConfig);
+
+        // TODO: fix webpack typings.
+        // tslint:disable-next-line:no-any
+        (webpackCompiler as any).inputFileSystem = new WebpackFileSystemHostAdapter(
+          new NodeJsSyncHost(),
```

Comment:
Shouldn't this be using the `this.context.host` instead of a new one? Or at least one based on the that one.

Reply:
Yes it should. Sorry I'll fix it. I tried it and it was failing, so decided to focus on one error at a time. Since all paths are absolute not to the Host but to the file system, that's a deeper change in general.